### PR TITLE
Overview of VLAN Trunking and Encapsulation

### PR DESCRIPTION
**Pull request by ShreyaAN10**
Requesting the team to review my article on VLAN Trunking and Encapsulation.
This article briefly introduces Virtual Local Area Networks(VLANs) and their requirements. It discusses VLAN trunking and VLAN encapsulation using IEEE 802.1Q standard. 
The folder contains 3 images: hero.jpg, native-vs-trunk.jpg, dot1q.jpg.
I also made some minor changes to my bio. I hope that is okay.

### CODE REVIEW

**Comment by hectorkambow** [comment]
Good afternoon @ShreyaAN10  - just started a preliminary review on your VLAN Trunking article - I noticed that there is also the index file of the VPN article.
Wanted to make sure we are keeping each article on a separate branch and separate PR to make merging easier. 👍

**Comment by ShreyaAN10** [comment]
Thanks for informing me about the earlier commits being in this branch @hectorkambow . I have removed them now. Hope everything else is alright.

**Comment by hectorkambow** [comment]
Good afternoon Dan 👋 @barthoda 🚀 - wanted to ask if you could give this article a tech review as well before we publish - whenever you have a moment.
Thank you

File: articles/vlan-trunking-and-encapsulation/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- Tagged or Trunk ports
+
+ *A network port is a communication endpoint a device uses to send and receive traffic.*
+
+**Native ports accept traffic from a single VLAN and are used to link end devices like routers, servers, terminals etc.** They do not need to accommodate an additional identification tag as they are involved in exchanging traffic with a single end device. Hence the name, **untagged ports**. They are also referred to as **access ports.**
+**Trunk ports are used to pass traffic from multiple VLANs and are used to link switches.** In order to distinguish between frames from different VLANs, trunk ports add identification tags to the frames.
+Hence the name, **tagged ports.**
+**A trunk is a point-to-point link between two network devices that carry more than one VLAN.**
+
+![native-vs-trunk](/engineering-education/vlan-trunking-and-encapsulation/native-vs-trunk.jpg)
```
Comment (hectorkambow) [inline articles/vlan-trunking-and-encapsulation/index.md:33]: @ShreyaAN10 wanted to ask if you could include the image sources underneath the images

Comment (ShreyaAN10) [reply articles/vlan-trunking-and-encapsulation/index.md:33]: @hectorkambow , I didn't pick these images from the web. I made them using diagrams.net

**Pull request merged**